.shipyard.e2e.ovn.yml

```diff
@@ -2,6 +2,7 @@
 cni: ovn
 submariner: true
 nodes: control-plane worker
+label-gateway: worker
```

Comment:
There no need for this then..

Reply:
@mkolesnik 
I know it's not needed.
I placed it here to serve as an example.
Otherwise, people will not be aware of that option.

Reply:
There are other options not in the files, best way to raise awareness is document on the website: https://submariner.io/development/shipyard/settings/

Reply:
Removed.
Will add it within the documentation.